src/unstract/adapters/llm/palm/src/static/json_schema.json

```
@@ -28,7 +28,7 @@
       "type": "number",
       "minimum": 0,
       "multipleOf": 1,
-      "title": "Number of Output Tokens",
+      "title": "Max output tokens",
```

Not setting any default here because, when we do not pass anything, llama-index automatically detects the max supported for the model and sets it accordingly.

---

Is this applicable only for Palm?

---

@gaya3-zipstack how will the user know this default value? What will happen if the user exceeds the max default?

---

Minimum `0` would not make sense. We could decide on the next best number.

---

> Is this applicable only for Palm?

Yes @Deepak-Kesavan

---

> @gaya3-zipstack how will the user know this default value? What will happen if the user exceeds the max default?

Ideally llama-index may throw error. However we have not extensively tested this. There are a few cases, where it silently passed and went through @chandrasekharan-zipstack